samtranslator/swagger/swagger.py

```diff
@@ -537,12 +537,18 @@ def set_path_default_authorizer(
             if normalized_method_name in SwaggerEditor._EXCLUDED_PATHS_FIELDS:
                 continue
             if add_default_auth_to_preflight or normalized_method_name != "options":
-                normalized_method_name = self._normalize_method_name(method_name)
+                SwaggerEditor.validate_is_dict(
+                    method,
+                    'Value of "{}" ({}) for path {} is not a valid dictionary.'.format(method_name, method, path),
+                )
                 # It is possible that the method could have two definitions in a Fn::If block.
                 for method_definition in self.get_method_contents(method):
 
                     SwaggerEditor.validate_is_dict(
-                        method_definition, "{} for path {} is not a valid dictionary.".format(method_definition, path)
+                        method_definition,
+                        'Value of "{}" ({}) for path {} is not a valid dictionary.'.format(
+                            method_name, method_definition, path
+                        ),
```

Comment:
what's the best way we can avoid having duplicate error messages (like this and the one couple lines above)? To handle this with path items, I created `validate_path_item_is_dict`, but I'm not sure if there's a better way of handling this (maybe a constant for the string, or a function to get the formatted string, or is it good as is?).